docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -338,6 +342,28 @@ header_name
   the request header is not present, the load balancer will use a random number as the hash,
   effectively making the load balancing policy random.
 
+.. _config_http_conn_man_route_table_route_add_req_headers:
+
+Adding custom request headers
+-----------------------------
+
+Custom request headers can be added to a request that matches a specific route. The headers are
```

Comment:
Can you link to this from the other request_headers_to_add? That way you don't have to duplicate the logic.